### PR TITLE
Section 2

### PR DESCRIPTION
**Pull request by chicco785**

### CODE REVIEW

**Comment by chicco785** [comment]
@martel-innovate/fiware-lab link to the build: https://join-fiware-lab-guide.readthedocs.io/en/section-2/

File: docs/2.policies/3.terms.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Terms & Conditions
+
+The Terms & Conditions is also updated following the last updated in the
+FIWARE ecosystem. This content makes reference to the use of the FIWARE
+Lab resources and the definition of the rights and obligations both in
```
Comment (taliaga) [inline docs/2.policies/3.terms.md]: ... and obligations **of** both FIWARE Lab and user**s** of these resources

File: docs/2.policies/3.terms.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+FIWARE Lab and user of these resources. The termination of these
+conditions and how to apply it besides with the intellectual property
```
Comment (taliaga) [inline docs/2.policies/3.terms.md]: This sentence is not clear. without a main verb. Also, "apply them" maybe?

File: docs/2.policies/3.terms.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+and proprietary rights. This content is reflected in the following URL:
+
+[*http://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/FIWARE\_LAB\_Terms\_and\_Conditions*](http://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/FIWARE_LAB_Terms_and_Conditions)
+
+The update of these Terms & Conditions will be notified to the users via
+email in order that they will be notified about the changes of them.
```
Comment (taliaga) [inline docs/2.policies/3.terms.md]: remove "of them" at the end.

**Review by taliaga — APPROVED** [review]

**Review by acca — APPROVED** [review]

**Pull request merged**